lib/app/admin/data-reset/test/remove-data-spec.js

```diff
@@ -0,0 +1,99 @@
+var sinon = require('sinon');
+require('sinon-as-promised');
+
+
+var remove = require('./../remove');
```

Comment:
When you're doing require() like this, you can leave out the `./` and just do `require('../remove')`. (The `./` is only required if it's in the same folder.)